src/bin/eolian/main.c

```diff
@@ -492,47 +493,99 @@ _write_deps(const Eolian_State *eos, const char *ofname, const char *ifname,
    return ret;
 }
 
-int
-main(int argc, char **argv)
+typedef struct _Cmd_Line_Options {
+   Eina_Bool scan_system;
+   Eina_List *includes;
+   int pret;
+   int gen_what;
+   char *basen;
+   char *outs[sizeof(_dexts) / sizeof(void *)];
+   const char *input;
+} Cmd_Line_Options;
+
+static int _process_options(int argc, char **argv, Cmd_Line_Options *options);
+
+static void _read_options_from_response_file(Cmd_Line_Options *options, const char *filename)
```

Comment:
:100: 
```suggestion
static void 
_read_options_from_response_file(Cmd_Line_Options *options, const char *filename)
```

Reply:
Oh wait NOT WITH THAT TRAILING WHITESPACE